src/main/scala/com/raphtory/algorithms/newer/OutDegreeAverage.scala

```diff
@@ -0,0 +1,26 @@
+package com.raphtory.algorithms.newer
+
+import com.raphtory.core.model.algorithm.{GraphAlgorithm, GraphPerspective, Row}
+
+class OutDegreeAverage(output:String) extends GraphAlgorithm {
+  override def algorithm(graph: GraphPerspective): Unit = {
+    var nodeCount = 0L
+    graph
+      .step({v => nodeCount += 1})
```

Comment:
You can go back to using the node count function as this should now be fixed and the above won't work